eiffel-vocabulary/EiffelSourceChangeCreatedEvent.md

```diff
@@ -0,0 +1,210 @@
+
+#### data.issues.type
+__Type:__ String  
+__Required:__ Yes  
+__Legal values:__ BUG, IMPROVEMENT, FEATURE, WORK_ITEM, REQUIREMENT, OTHER  
+__Description:__ The type of issue.
+  
+#### data.issues.tracker
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The name of the issue tracker. This can unfortunately not be standardized, and is therefore context sensitive: though some trackers and ALM tools are more popular than others, an exhaustive enumeration is impossible, particularly when considering company specific internal solutions. Consequently one should not rely on the name as the primary method of retrieval, but rather __data.issues.uri__. __data.issues.tracker__ together with __data.issues.id__ is still useful for analysis and traceability, however, as long as it can be correctly interpreted.
+
+#### data.issues.id
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The identity of the issue. This is tracker dependent - most trackers have their own issue naming schemes.
+
+#### data.issues.uri
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The URI of the issue.
+
+#### data.issues.transition
+__Type:__ String  
+__Required:__ Yes  
+__Legal values:__ RESOLVED, PARTIAL, REMOVED  
+__Description:__ The new state of the issue: does the change resolve it, partially resolve it or remove it?
+
+### data.gitIdentifier
+__Type:__ Object  
+__Required:__ No  
+__Description:__ Identifier of a Git change.
+
+#### data.gitIdentifier.commitId
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The commit identity (hash) of the change.
+
+#### data.gitIdentifier.branch  
+__Type:__ String  
+__Required:__ No  
+__Description:__ The name of the branch where the change was made.
+
+#### data.gitIdentifier.repoName  
+__Type:__ String  
+__Required:__ No  
+__Description:__ The name of the repository containing the change.
+
+#### data.gitIdentifier.repoUri  
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The URI of the repository containing the change.
+
+### data.svnIdentifier
+__Type:__ Object  
+__Required:__ No  
+__Description:__ Identifier of a Subversion change.
+
+#### data.svnIdentifier.revision
+__Type:__ Integer  
+__Required:__ Yes  
+__Description:__ The revision of the change.
+
+#### data.svnIdentifier.directory  
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The directory (branch/tag) of the change.
+
+#### data.svnIdentifier.repoName  
+__Type:__ String  
+__Required:__ No  
+__Description:__ The name of the repository containing the change.
+
+#### data.svnIdentifier.repoUri  
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The URI of the repository containing the change.
+
+### data.ccCompositeIdentifier
+__Type:__ Object  
+__Required:__ No  
+__Description:__ Identifier of a composite ClearCase change – in other words, not single file commit, but analogous of repository-wide commits of e.g. SVN or Git.
+
+#### data.ccCompositeIdentifier.vob
+__Type:__ String  
```

Comment:
Having bounced this off of a few people and given the fact that a configSpec can and often does deal with multiple VOBs, I'll push a commit changing this to a list.